tendrl/integrations/ceph/flows/add_osds/__init__.py

```diff
@@ -13,6 +17,64 @@ def run(self):
         if integration_id is None:
             raise FlowExecutionFailedError("TendrlContext.integration_id cannot be empty")
 
+        ssh_job_ids = []
+        ssh_setup_script = NS.ceph_provisioner.get_plugin().setup()
+        if len(self.parameters["Cluster.node_configuration"].keys()) > 0:
+            for node_id in self.parameters["Cluster.node_configuration"].keys():
```

Comment:
Will remove the if check and `Cluster.node_configuration` is mandatory parameter for the flow.